pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -817,6 +825,7 @@ private void ensureAllNodesAreInWhitelist(
               new MiningParameters(coinbase, minTransactionGasPrice, extraData, isMiningEnabled))
           .maxPendingTransactions(txPoolMaxSize)
           .pendingTransactionRetentionPeriod(pendingTxRetentionPeriod)
+          .txMessageKeepAliveSeconds(txMessageKeepAliveSeconds)
```

Comment:
(Optional) Might be time for a `TransactionPoolConfiguration` object.

Reply:
I will do this in a follow up pr.